src/NuGet.Core/NuGet.Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using NuGet.Common;
+
+namespace NuGet.Commands
+{
+    public static class ConfigPathsRunner
+    {
+        public static void Run(ConfigPathsArgs args, Func<ILogger> getLogger)
+        {
+            var settings = RunnerHelper.GetSettingsFromDirectory(args.WorkingDirectory);
+
+            if (settings != null)
+            {
+                var filePaths = settings.GetConfigFilePaths();
+                foreach (var filePath in filePaths)
+                {
+                    getLogger().LogMinimal(filePath);
+                }
```

Comment:
```suggestion
                ILogger logger = getLogger();
                foreach (var filePath in filePaths)
                {
                    logger.LogMinimal(filePath);
                }
```

```suggestion
                    getLogger().LogMinimal(filePath);
```
I'm not sure the precedent in our code, but it seems like it would be more efficient and cleaner to call `getLogger` once and then call `LogMinimal()` multiple times instead of calling `getLogger().LogMinimal()` over and over.